cmd/rigs/cmd/publish.go

```diff
@@ -23,17 +26,18 @@ import (
 var (
 	_db *sql.DB
 
-	ethClient    *ethclient.Client
-	dirPublisher *dirpublisher.DirPublisher
-	store        storage.Store
-	tblClient    *client.Client
-	chain        chains.Chain
+	ethClient *ethclient.Client
+	pub       *publisher.Publisher
+	store     storage.Store
+	tblClient *client.Client
+	chain     chains.Chain
 )
 
 func init() {
 	rootCmd.AddCommand(publishCmd)
 
 	publishCmd.PersistentFlags().String("nft-storage-key", "", "api key for nft.storage")
+	publishCmd.PersistentFlags().String("web3-storage-key", "", "api key for web3.storage")
```

Comment:
Added the option to use web3.storage in addition to nft.storage for storing imagery. There is a storage interface and implementations for both services.